libs/coin-modules/coin-aptos/src/__tests__/api/index.test.ts

```diff
@@ -9,16 +9,15 @@ import {
   Serializable,
   post,
 } from "@aptos-labs/ts-sdk";
-import network from "@ledgerhq/live-network/network";
+import network from "@ledgerhq/live-network";
 import BigNumber from "bignumber.js";
-import { AptosAPI } from ".";
-import { Account } from "../../../e2e/enum/Account";
+import { AptosAPI } from "../../api";
 
 jest.mock("@aptos-labs/ts-sdk");
 jest.mock("@apollo/client");
-let mockedAptos;
-let mockedApolloClient;
-let mockedPost;
+let mockedAptos: jest.Mocked<any>;
```

Comment:
is it possible to inject the type for the generic Mocked ?

Reply:
If I do that, I get a `Type 'MockedObjectDeep<typeof Aptos>' is not assignable to type 'Mocked<Aptos>'`
If I do 
```suggestion
let mockedAptos: jest.MockedObjectDeep<typeof Aptos>;
```
I get `Type '{ getAccountInfo: Mock<any, any, any>; }' is missing the following properties from type 'Aptos': config, account, ans, coin, and 111 more`

Basically I'm not able to mock it without adding all of its properties. 

Let me know if there's a better way to do it, I also hate to use `any`.

Reply:
did you try with `jest.MockedObjectDeep<Partial<typeof Aptos>>`

Reply:
With Partial, I get an error on line 71:
```
mockedAptos.mockImplementation(() => ({
  getAccountInfo: mockGetAccountInfo,
}));
```
`mockImplementation does not exist`

And if I remove `mockedAptos = jest.mocked(Aptos);` from line 27, and add the following inside the tests
```
mockedAptos = jest.mocked(Aptos).mockImplementation(() => ({
  getAccountInfo: mockGetAccountInfo,
}));

```
I get the same error as before:
`Type '{ getAccountInfo: Mock<any, any, any>; }' is missing the following properties from type 'Aptos': config, account, ans, coin, and 111 more`

Reply:
ok ! no worries, i'll resolve this

Reply:
@themooneer can you describe what is that you plan to do? Thanks